src/lightning/__init__.py

```diff
@@ -18,6 +18,7 @@
 from lightning.app import storage  # noqa: E402
 from lightning.app.core.app import LightningApp  # noqa: E402
 from lightning.app.core.flow import LightningFlow  # noqa: E402
+from lightning.app.core.plugin import LightningPlugin  # noqa: E402
```

Comment:
Find is confusing with PL precisions plugins... :thinking:

Reply:
Removed top-level import, so it will be `L.app.LightningPlugin`